src/resources/MachineLearning/SESConfiguration/SESConfigurationInterfaces.ts

```diff
@@ -0,0 +1,50 @@
+import {DocumentRequirementStatus} from '../Document.js';
+import {FilterConditions} from '../FilterConditions.js';
+
+export interface SESDocumentGroupPreviewParams {
+    /**
+     * The sources to consider.
+     */
+    sources: string[];
+    /**
+     * An array of filtering conditions.
+     */
+    filterConditions: FilterConditions[];
+    /**
+     * The query that determines the documents to extract
+     */
+    advancedQuery: string;
+}
+
+export interface SESDocumentGroupPreview {
+    /**
+     * The query that was used to fetch document information.
+     *
+     * @Example @source==("Salesforce Notifier")
+     */
+    query: string;
+    /**
+     * The total number of documents in the selected sources.
+     */
+    numberOfDocumentsInSources: number;
+    /**
+     * The number of documents that are candidates for learning.
+     */
+    numberOfValidDocuments: number;
```

Comment:
What's the difference between this and numberOfDocumentsInSourcesMatchingFiltersWithPermanentId?

Reply:
The `numberOfValidDocuments` also matches on the `language` being english

Reply:
OK, I think this could be clarified in the documentation, but it's not the end of the world.